x/confidentialtransfers/types/ct_account.go

```diff
@@ -0,0 +1,79 @@
+package types
+
+import (
+	"github.com/coinbase/kryptology/pkg/core/curves"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/sei-protocol/sei-cryptography/pkg/encryption/elgamal"
+)
+
+func (m *CtAccount) ValidateBasic() error {
+	account, err := m.FromProto()
```

Comment:
I have moved away from converting stuff in ValidateBasic btw, just checking if fields are there.